packages/scripts/src/monitoring-dvm2.0/voterStakes.ts

```diff
@@ -0,0 +1,62 @@
+// This script verifies the correct functioning of the VotingV2 contract. In particular, it verifies that, once all the
+// voters are processed by the updateTrackers function, the balance of the VotingV2 contract in VotingTokens is equal to
+// the sum of all users' stakes and pendingUnstakes. Otherwise it throws an error.
+// It can be run on a mainnet or goerli fork, with the following command and environment variables:
+// Or a goerli fork with:
+// CUSTOM_NODE_URL=https://<goerli OR mainnet>.infura.io/v3/<YOUR-INFURA-KEY> \
+// yarn hardhat run ./src/monitoring-dvm2.0/voterStakes.ts
+
+const hre = require("hardhat");
+import { bigNumberAbsDiff } from "../utils/utils";
+import {
+  getNumberSlashedEvents,
+  getUniqueVoters,
+  getVotingContracts,
+  updateTrackers,
+  votingV2VotingBalanceWithoutExternalTransfers,
+} from "./common";
+const { ethers } = hre;
+
+async function main() {
+  const { votingV2, votingToken } = await getVotingContracts();
+
+  const uniqueVoters = await getUniqueVoters(votingV2);
+
+  // Update trackers for all voters
+  await updateTrackers(votingV2, uniqueVoters);
+
+  // get voter stakes for each voter
+  const voterStakes = await Promise.all(
+    uniqueVoters.map(async (voter) => {
+      const stake = await votingV2.voterStakes(voter);
+      return stake.stake.add(stake.pendingUnstake);
+    })
+  );
+
+  const sumStakes = voterStakes.reduce((a, b) => a.add(b), ethers.BigNumber.from(0));
+
+  const numberSlashedEvents = await getNumberSlashedEvents(votingV2);
+  const votingV2BalanceWithoutExternalTransfers = await votingV2VotingBalanceWithoutExternalTransfers(
+    votingToken,
+    votingV2
+  );
```

Comment:
This was super tricky to implement +1 -- I'm guessing this will save us a lot of on-call pain so we don't get paged when somebody accidentally sends funds to the contract.